pkgs/servers/db-rest/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, nodejs_18
+, nix-update-script
+}:
+let
+  buildNpmPackage' = buildNpmPackage.override { nodejs = nodejs_18; };
+in buildNpmPackage' rec {
```

```suggestion
buildNpmPackage rec {
```

You can actually specify `nodejs` as an argument to `buildNpmPackage` now as of #265171, so move this to the `buildNpmPackage` args, please